src/ChaiWrapper.js

```diff
@@ -151,16 +151,20 @@ export default class ChaiWrapper {
 
     return function (arg1, arg2) {
       const wrapper = wrap(flag(this, 'object'))
-
-      assertion.call(this, {
+      const config = {
         markup: () => debug(wrapper),
         sig: inspect(wrapper),
         wrapper,
         arg1,
-        arg2,
         flag,
         inspect
-      })
+      }
+
+      if (arguments.length > 1) {
```

Comment:
Can we leave a note here explaining why checking `arg2 === undefined` is not sufficient